README.md

```diff
@@ -1,3 +1,4 @@
 # Hej, världen!
 This is octo-cat trolling.
+I'm trollin' so much I add a 2nd line!
```

Comment:
What is this? I think this is **not** very suitable in a readme. 
It would be better if it said:
`I'm trolling just a litte bit.`